src/schema/schema_worker.c

```diff
@@ -68,29 +68,56 @@ __schema_tiered_worker(WT_SESSION_IMPL *session, const char *uri,
 
 /*
  * __schema_layered_worker_verify --
- *     Run a schema worker operation (which is verification) on the stable table; ingest is not
- *     supported currently.
+ *     Run a schema worker operation (which is verification) on the layered table.
  *
- * FIXME-WT-15047: Implement ingest table verification (and update the function description)
+ * FIXME-WT-15047: Implement ingest table verification on followers.
  */
 static int
 __schema_layered_worker_verify(WT_SESSION_IMPL *session, const char *uri,
   int (*file_func)(WT_SESSION_IMPL *, const char *[]),
   int (*name_func)(WT_SESSION_IMPL *, const char *, bool *), const char *cfg[], uint32_t open_flags)
 {
+    WT_CONNECTION_IMPL *conn;
     WT_DECL_RET;
 
+    conn = S2C(session);
+
     WT_RET(__wt_session_get_dhandle(session, uri, NULL, NULL, open_flags));
     WT_LAYERED_TABLE *layered = (WT_LAYERED_TABLE *)session->dhandle;
     const char *stable_uri = layered->stable_uri;
+    const char *ingest_uri = layered->ingest_uri;
 
     WT_ASSERT(session, stable_uri != NULL);
+    WT_ASSERT(session, ingest_uri != NULL);
     WT_ASSERT(session, file_func == __wt_verify);
 
+    /*
+     * Verifying stable tables of layered tables uses the existing verify logic.
+     * Ingest tables, however, require special handling:
+     * - On leader: ingest must always be empty/no-op.
+     * - On followers: ingest contains recent oplog updates layered over the stable table.
+     */
+
+    if (conn->layered_table_manager.leader) {
+        /*
+         * On leader, if verifying ingest returns EBUSY, it means ingest is not empty (dirty
+         * content, open cursors, or checkpoints present), which is an invalid state.
+         */
+        WT_WITHOUT_DHANDLE(session,
+          ret = __wt_schema_worker(session, ingest_uri, file_func, name_func, cfg, open_flags));
+        if (ret == EBUSY)
+            WT_ERR_MSG(session, WT_ERROR,
+              "Verify (layered): %s ingest table on leader cannot be verified. Returned EBUSY, "
+              "this is an invalid access pattern",
```

Comment:
Can we make this error message more informative for users? I would propose something like: "Ingest contains dirty content or cursors opened which is an invalid state". WDYT?

Reply:
I've meant replacing "Verify (layered): %s ingest table on leader cannot be verified." with "Ingest contains dirty content or open cursors, which is an invalid state." since I don't think that we should report that "ingest table on leader cannot be verified.".

I think it should be treated as we verified the table and found the issue instead of not being able to verify the table.